gensrc/thrift/Types.thrift

```diff
@@ -135,6 +135,9 @@ struct TTypeNode {
 
     // only used for structs; has struct_fields.size() corresponding child types
     3: optional list<TStructField> struct_fields
+
+    // only used for structs; for output use
+    4: optional bool is_named
```

Comment:
do we have to put the `is_named` info in TTypeNode struct? I think whether has name, not a type info

Reply:
Now the implemente has a name, so we need the type info to describe it's anonymous struct. of course, the field is useless now, it's mostly used to control whether to print the subfiled name in future. if you want use empty fields name to describe anonymous struct, it's means we should handle anonymous struct in all struct's code. BTW, use `is_named` for better compatibility

Reply:
yeah, keep same with trino. BTW, I think we can modify the name visibility by current impelement，this design is mainly to solve BE need to deal with anonymous struct/name struct